android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -1087,6 +1081,7 @@ protected void onStart()
     mPlacePageController.onActivityStarted(this);
     mSearchController.attach(this);
     MwmApplication.backgroundTracker(getActivity()).addListener(this);
+    updateViewsInsets();
```

Comment:
I took a close look at the fix we merged yesterday. I think that calling `updateViewsInsets()` from `onSafeCreate` should be more than enough. Please take a look at https://github.com/organicmaps/organicmaps/pull/3453

Reply:
Either @arnaudvergnet should take your changes from #3453, or you should take @arnaudvergnet changes from this PR to properly test the final version before the merge.

Please also take into account my comments: https://github.com/organicmaps/organicmaps/pull/3451#issuecomment-1250397837

Reply:
Maybe moving @rtsisyk changes here would be easier so we can continue working on other improvements.

Reply:
Yes, please feel free to amalgamate my PR.